js/i18n.js

```diff
@@ -73,6 +73,26 @@ function searchForResources(search_url) {
       var time = getSingleItem(params, 'time');
       var manifest = getSingleItem(params, 'manifest');
       var languages = getArrayItem(params, 'lc');
+      
+      var searchFieldArr = [];
+      if(full_text) searchFieldArr.push(full_text);
+      // if(repo_name) searchFieldArr.push('repo:'+repo_name);
```

Comment:
Ok, commented out these lines (didn't want to have to type them out again later), but still think that these wouldn't have a been a problem since the URL has to be build from the search field originally and not vice versa.